packages/headless/src/api/service/case-assist/get-case-classifications/get-case-classifications-request.ts

```diff
@@ -14,7 +14,7 @@ import {
 
 export type GetCaseClassificationsRequest = BaseParam &
   CaseAssistIdParam &
-  VisitorIDParam &
+  ClientIDParam &
```

Comment:
Isn't removing `VisitorIDParam` from the type a breaking change? An error would be raised for the snippet below.

```typescript
buildGetCaseClassificationsRequest({ ..., visitorId: 'some-id' });
```

Reply:
Given the client is not exported I don't think there can be any instance of such usage outside of Headless. At least I couldn't find any hit of it.

Reply:
I could easily leave it in though if we do find it is breaking.

Reply:
> Given the client is not exported I don't think there can be any instance of such usage outside of Headless.

If it is not exported by the Headless bundle, then we're good.

Reply:
As far as I can tell, this is only used within Headless itself, so it should be okay 🙂